src/tekton/tektonitem.ts

```diff
@@ -89,7 +96,15 @@ export abstract class TektonItem {
   }
 
   static openInEditor(context: TektonNode): void {
-    TektonItem.loadTektonResource(context.contextValue, context.getName());
+    let name = context.getName();
+    if (context.contextValue === 'tr') {
```

Comment:
It would be better to use `ContextType .TR` to compare instead of hardcoded value